code/modules/martial_arts/cqc.dm

```diff
@@ -14,10 +14,14 @@
 
 /datum/martial_art/cqc/under_siege/teach(mob/living/carbon/human/H, make_temporary)
 	RegisterSignal(H, COMSIG_AREA_ENTERED, PROC_REF(kitchen_check))
+	ADD_TRAIT(H.mind, TRAIT_EXPERT_CHEF, ROUNDSTART_TRAIT) // cqc makes it a lot easier to track if someone REALLY is a chef :)
+	add_verb(H, /mob/living/carbon/human/proc/expert_chef_knowledge)
 	return ..()
 
 /datum/martial_art/cqc/under_siege/remove(mob/living/carbon/human/H)
 	UnregisterSignal(H, COMSIG_AREA_ENTERED)
+	REMOVE_TRAIT(H.mind, TRAIT_EXPERT_CHEF, ROUNDSTART_TRAIT)
+	remove_verb(H, /mob/living/carbon/human/proc/expert_chef_knowledge)
```

Comment:
Don't tie this to CQC, please. Add the trait here:
https://github.com/ParadiseSS13/Paradise/blob/de891f6de830fd05c1c6bcaa664f5202617bbee0/code/game/jobs/job/support.dm#L239